### ISSUE TITLE
Request for (more) adjustable sidebar layout and default settings to enhance focus and usability

### ISSUE THREAD

**Issue by DucklearExplosion** [ls1intum/Artemis#10066]
### Is your feature request related to a problem?

As a student I find it frustrating and distracting that half of my screen by default gets taken up by side bars, which don't really give me any helpful additional information while doing the exercise, like the communication or exercise bar. Especially on smaller screens it leads to a lot less space for the most important part of the page - the exercise description and sometimes diagrams related to it. Having to toggle all the side bars every time I start working on a new task just to be able to see a diagram properly or to be able to focus on reading the task description is quite frustrating and the overall default layout with all sidebars being extended looks quite cluttered and overwhelming.
![image](https://github.com/user-attachments/assets/6aa85db6-7478-4767-abd8-ba26688d99b4)
(As can be seen in the picture, at least a third of my screen gets taken up by sidebars which leads to the diagram being barely visible in the view one sees when clicking on a course to do the tasks)

### Describe the solution you'd like

It would be great if the side bars were hidden by default and if we had the option to entirely disable/hide the communication bar from our student view when trying to focus on an exercise as it would reduce distractions.

**Comment by lana-ati** [comment]
I agree, it makes sense for at least the right sidebar to be hidden by default as it makes it difficult to read the content at times.

**Comment by Jess-hub09** [comment]
I think that this request is actually outdated, Artemis already supports collapsing the sidebars, and once collapsed, the exercise description and diagrams are fully visible without the UI feeling cluttered. You can also permanently hide the communication sidebar during work sessions. So the functionality the issue describes is already available in the current version.

<img width="1889" height="880" alt="Image" src="https://github.com/user-attachments/assets/78bc00a3-952b-42f0-8b13-2bc4a7d2f88f" />

<img width="1912" height="864" alt="Image" src="https://github.com/user-attachments/assets/8c75a088-2ce0-4029-9eec-b83e6655f3bc" />
